app/views/refinery/acts_as_indexed/admin/_search.html.erb

```diff
@@ -1,5 +1,10 @@
 <form method='GET' action='<%= url %>' class='search_form'>
-  <%= text_field :search, nil, :id => 'search', :type => 'search', :name => 'search', :value => params[:search], :title => t('search_input_notice', :scope => 'refinery.admin.search') %>
+  <%= text_field :search, nil, :id => 'search', :type => 'search',
+    :name => 'search',
+    :value => params[:search],
+    :title =>       t('search_input_help', :scope => 'refinery.admin.search', :query => h(params[:search])).html_safe,
```

Comment:
Actually for this one, I think I could probably do an adequate job on locales that use the same alphabet, as one of the changes is a single word and the new key is a copy of the old key.